### PR TITLE
Added Dockerfile for Remote Cargo

### PR DESCRIPTION
**Pull request by yangwao**
Adding Dockerfile, dockerignore and docker-compose when using compiled edgeware binary on Linux to use with Remote Cargo on OSX

### CODE REVIEW

**Comment by yangwao** [comment]
Ok, I've packaged symlinks to one script, but it's **not perfect**, but it works, we can later iterate this.

PIA is multiple .dockerignore and within folder didn't worked for me ;( https://www.thetopsites.net/article/53524119.shtml

**Comment by yangwao** [comment]
I just realized it would be good to be in develop branch as well.

**Comment by drewstone** [comment]
can we add some documentation to the README on this? how to run it and what it is used for?

File: README.md [hunk 1]
```diff
@@ -73,3 +73,16 @@ To start up the Edgeware node and connect to the Beresheet testnet, run:
 ```
 ./target/release/edgeware --chain=beresheet --name <INSERT_NAME>
 ```
+
+### Docker
+
```
Comment (drewstone) [inline README.md:78]: this is a little sparse as to what the remote-cargo system is for. Some background info would be great.

**Comment by yangwao** [comment]
Sure, I just had previously to publish an article so I can link it now. Making PR to fix it

**Pull request merged**